lib/model/folder_summary.go

```diff
@@ -77,28 +78,36 @@ func (c *folderSummaryService) String() string {
 func (c *folderSummaryService) Summary(folder string) (map[string]interface{}, error) {
 	var res = make(map[string]interface{})
 
-	snap, err := c.model.DBSnapshot(folder)
-	if err != nil {
-		return nil, err
-	}
-
+	var local, global, need, ro db.Counts
+	var ourSeq, remoteSeq int64
 	errors, err := c.model.FolderErrors(folder)
+	if err == nil {
+		var snap *db.Snapshot
+		if snap, err = c.model.DBSnapshot(folder); err == nil {
```

Comment:
release dat snap

Reply:
Also, DBSnapshot still returns `errFolderMissing` for non-started folders does it not?

Reply:
The snap is released if its acquired.

Reply:
Now it is, yes, but it wasn't when I commented. :)